src/main/java/com/owncloud/android/lib/common/OwnCloudBasicCredentials.java

```diff
@@ -71,4 +73,63 @@ public boolean authTokenExpires() {
     public String toOkHttpCredentials() {
         return Credentials.basic(username, authToken);
     }
+
+    /*
+     * Autogenerated equals and hash
```

Comment:
Lombok, see above

Reply:
@AndyScherzinger Thanks, I remved explicit implementation and enabled Lombok generator.